### DISCUSSION TITLE
设置了‘baseLang’为中文后评论功能的部分显示仍为英文

### DISCUSSION THREAD

**Discussion by tenngoxars** [vuepress-theme-hope/vuepress-theme-hope#539] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/22515339/103873900-7424f000-510b-11eb-9d8a-bb1893ecb67e.png)
请问这里可能是哪里没有设置对呢？

**Comment by Mister-Hope** [comment]
I will have a look whether there is anything wrong.

**Comment by Mister-Hope** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/33315834/103904593-6f2a6580-5138-11eb-8c2f-2b37e1c7a4df.png)

I tried setting `themeConfig.baseLang` to `zh-CN` and it's working fine.

Can you give me a reproduction repo link or just your repo? Thanks.

**Reply by tenngoxars** [reply]
是我的问题，没有放在`themeConfig`下，现在已正常，非常感谢~
